wow/github-workflow.md

```diff
@@ -0,0 +1,274 @@
+# Github Workflow
+
+## Overview
+
+The workflow is based on the understanding that:
+
+- A project is forked from Open Source project to nordix
+- A local copy is cloned from nordix
+
+We refer to the Open Source project repo as `upstream`, `upstream/main`
+and to the forked repo in the Nordix organization as `origin`,
+`origin/main`
+
+Main branch is called `main`, containing the latest stable release.
+
+Feature development and bug fixing are done in topic branches, branched
+of `main` branch. Upon completion and code review, topic branch is
+merged into `upstream` branch.
+
+## Branches
+
+### Topic branches (features and bug fixes)
+
+Topic branches need to be branched off `main` and named
+`type/name-username`, where  type is `feature` or `fix` and `username`
```

Comment:
Could we update this? I would rather see a system like `user/topic` or `user/type-topic` for the branch name.
The reason I like this style is that it is easier to identify the user when it is at the start of the name and easy to find all your own branches with autocomplete.
We are anyway not following the naming scheme so we might as well change it :upside_down_face:

Reply:
With this will lose the history what if we merge this then made a PR to change this

Reply:
Sure we can update it later